charts/shoot-core/charts/monitoring/charts/blackbox-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: apiserver-connectivity-check
```

Comment:
I would name this `blackbox-exporter` as in the future we might add more checks that go way beyond apiserver only.

Same goes for the `Service`

Reply:
Renamed to blackbox-exporter.